Tasks/UseRubyV1/installer.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+import * as task from "azure-pipelines-task-lib/task";
+import * as tool from "azure-pipelines-tool-lib/tool"
+
+enum Platform {
+    Windows,
+    MacOS,
+    Linux
+}
+
+function getPlatform(): Platform {
```

Comment:
This exact function/enum [exists in the task-lib](https://github.com/Microsoft/azure-pipelines-task-lib/blob/e5a622a92dd95b77b30daf5ce8ae8095301d2a2c/node/task.ts#L588) (it started in the tasks repo but so many tasks were using it that we added it).

Reply:
👍 Updated to use task-lib here